cmd/config-translator/translator.go

```diff
@@ -47,6 +46,7 @@ func initFlags() {
 		if err != nil {
 			log.Fatalf("E! Failed to open common-config file %s with error: %v", *inputConfig, err)
 		}
+		defer f.Close()
```

Comment:
Unrelated change.
It is usually a big deal when you forget to close files... But this `config-translator` program is short lived and when it exits everything is released anyway...